core/blockchain.go

```diff
@@ -72,14 +72,11 @@ var (
 	storageUpdateTimer = metrics.NewRegisteredResettingTimer("chain/storage/updates", nil)
 	storageCommitTimer = metrics.NewRegisteredResettingTimer("chain/storage/commits", nil)
 
-	snapshotAccountReadTimer = metrics.NewRegisteredResettingTimer("chain/snapshot/account/reads", nil)
-	snapshotStorageReadTimer = metrics.NewRegisteredResettingTimer("chain/snapshot/storage/reads", nil)
-	snapshotCommitTimer      = metrics.NewRegisteredResettingTimer("chain/snapshot/commits", nil)
```

Comment:
TODO: lift this to PR-description, (also to get it mentioned in future release notes)
----------------

This PR removes the following metrics 
- `chain/snapshot/account/reads`
- `chain/snapshot/storage/reads`
- `chain/snapshot/snapshot/commits`

It adds a new metric, `chain/triedb/commits`.

Reply:
done, 

`chain/snapshot/snapshot/commits` is not removed
`chain/triedb/commits` was existent before